source/Types.cpp

```diff
@@ -51,53 +51,53 @@ namespace Aws
             return aws_byte_cursor_from_array(array, len);
         }
 
-        Vector<uint8_t> Base64Decode(const String &decode)
+        Vector<uint8_t> Base64Decode(const String &decode) noexcept
```

Comment:
> Exceptions
>Calls to Allocator::allocate may throw. 

https://en.cppreference.com/w/cpp/container/vector/vector

While by default it is built is exceptions disabled, idk what is the point of marking methods as noexcept when the inner code can throw (if exceptions are enabled).
I.e. technically `noexcept` is incorrect here.

Reply:
well, on another thought, CRT uses Vector with a custom allocator from CRT that does not throw